release-content/0.14/release-notes/12013_Upstreaming_bevy_color.md

```diff
@@ -5,7 +5,7 @@ and offers a wide range of fully documented operations on and conversions betwee
 Critically, color mixing / blending is now supported: perfect for procedurally generating color palettes and working with animations.
 
 ```rust
-use bevy_color::{palettes::tailwind, Mix};
+use bevy_color::*;
```

Comment:
```suggestion
use bevy_color::prelude::*;
```

Nit: `bevy_color::*` would work, but we have a prelude so we might as well show it off.